src/components/SettingsPage/CleanButton.js

```diff
@@ -33,11 +31,15 @@ class CleanButton extends PureComponent<Props, State> {
 
   close = () => this.setState({ opened: false })
 
+  actionTriggered = false
   action = async () => {
-    this.props.cleanAccountsCache()
-    await delay(500)
-    db.cleanCache()
-    remote.getCurrentWindow().webContents.reload()
+    if (this.actionTriggered) return
```

Comment:
I think it would be better to put `isLoading` on corresponding `Button` (which disable it + display a spinner) like it's [done in ConfirmModal](https://github.com/LedgerHQ/ledger-live-desktop/blob/37ee4525915fb0926a1826b6125e498bfdf1a9d5/src/components/base/Modal/ConfirmModal.js#L81) because it provide feedback to user.

Anyway, I understand we have to deliver fast this release so it can wait but those kind of micro interactions are :heart: